### ISSUE TITLE
The parameter 'af'

### ISSUE THREAD

**Issue by GroundZeros** [iBelieveCJM/pseudo_label-pytorch#5]
https://github.com/iBelieveCJM/pseudo_label-pytorch/blob/66821d33d8ddaf34563c37ed4138999b17e0a4ef/experiments/cifar10_test.py#L55

'af' = 3 in the paper  _Pseudo-Label The Simple and Efficient Semi-Supervised Learning Method for Deep Neural Networks_  ,
I would like to know whether 0.3 in here help model perform better than 3 in paper.
